homeassistant/components/unifi_direct/device_tracker.py

```diff
@@ -121,7 +121,9 @@ def _get_update(self):
 
 def _response_to_json(response):
     try:
-        json_response = json.loads(str(response)[31:-1].replace("\\", ""))
+        if (str(response)[31:-1].find(UNIFI_SSID_TABLE) == -1):
+            return {}
+        json_response = json.loads(str(response)[31:-1].replace("\\\\", "\\").replace("\\'", "'"))
```

Comment:
Please extract the device specific interface code and data parsing to a standalone library published on PyPI.

https://developers.home-assistant.io/docs/creating_component_code_review#4-communication-with-devicesservices